### PR TITLE
feat(webhook): add logic for alert parsing

### PR DESCRIPTION
**Pull request by pietro-tota**
Depends on https://github.com/pagopa/pagopa-infra/pull/1293

#### List of Changes

- Add logic to parse alert: this logic parses input alert web-hook Grafana requests and translate each received alert into EventHub event
- Add endpoint for app info: this endpoint is used as liveness/readiness endpoint for k8s container
- Upgrade function runtime to java 17

#### Motivation and Context
Those modifications implements logic for parse input web-hook Grafana requests and translate it into events so other services can consume those events and take required actions

#### How Has This Been Tested?
Junit tests + test in DEV env

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/update_code.yml [hunk 1]
```diff
@@ -25,10 +25,10 @@ jobs:
         run: hub pr checkout ${{ github.event.issue.number }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Set up JDK 11
+      - name: Set up JDK 17
         uses: actions/setup-java@v1
```
Comment (FedericoRuzzier) [inline .github/workflows/update_code.yml:29]: Why java 17? Aren't we using 11?

File: .java-version [hunk 2]
```diff
@@ -1 +1 @@
-11
+17
```
Comment (FedericoRuzzier) [inline .java-version:1]: See previous comment

File: .github/workflows/update_code.yml (hunk 1 above)
Comment (pietro-tota) [reply .github/workflows/update_code.yml:29]: I upgrade the function runtime to java 17

File: .java-version (hunk 2 above)
Comment (pietro-tota) [reply .java-version:1]: I upgrade the function runtime to java 17

**Review by FedericoRuzzier — APPROVED** [review]
LGTM 🥇

**Pull request merged**